src/dev/flang/ast/AbstractFeature.java

```diff
@@ -1140,21 +1140,6 @@ public boolean definesType()
   }
 
 
-  /**
-   * Are calls to this feature performed using dynamic binding?
-   */
-  public boolean isDynamic()
```

Comment:
It makes sense to me intuitively that this is removed since IMHO it should not be for a feature to decide _how_ it is being called.